{"schemaVersion":{"minor":3,"major":0,"patch":0},"variants":[{"paths":["\/documentation\/mdots\/autherror\/errordescription"],"traits":[{"interfaceLanguage":"swift"}]}],"sections":[],"identifier":{"url":"doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots\/AuthError\/errorDescription","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots","doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots\/AuthError"]]},"metadata":{"fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"errorDescription"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":"?"}],"symbolKind":"property","modules":[{"name":"MDots"}],"externalID":"s:5MDots9AuthErrorO16errorDescriptionSSSgvp","title":"errorDescription","roleHeading":"Instance Property","role":"symbol"},"abstract":[{"type":"text","text":"A description of the error, suitable for displaying to the user."}],"kind":"symbol","primaryContentSections":[{"declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"text":"errorDescription","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":"? { ","kind":"text"},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"platforms":["iOS"],"languages":["swift"]}],"kind":"declarations"},{"kind":"content","content":[{"text":"Discussion","level":2,"type":"heading","anchor":"discussion"},{"inlineContent":[{"text":"This property provides a localized description for each error case, making it easier to","type":"text"},{"type":"text","text":" "},{"text":"understand what went wrong during the authentication process.","type":"text"}],"type":"paragraph"}]}],"references":{"doc://com.estelapasjak.Mdots/documentation/MDots/AuthError/errorDescription":{"kind":"symbol","title":"errorDescription","abstract":[{"text":"A description of the error, suitable for displaying to the user.","type":"text"}],"fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"text":"?","kind":"text"}],"url":"\/documentation\/mdots\/autherror\/errordescription","type":"topic","identifier":"doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots\/AuthError\/errorDescription","role":"symbol"},"doc://com.estelapasjak.Mdots/documentation/MDots":{"identifier":"doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots","title":"MDots","kind":"symbol","role":"collection","type":"topic","url":"\/documentation\/mdots","abstract":[],"images":[{"identifier":"new-icon.png","type":"icon"}]},"new-icon.png":{"variants":[{"traits":["1x","light"],"url":"\/images\/new-icon.png"}],"identifier":"new-icon.png","type":"image","alt":"The logo representing the MDot sensor application."},"doc://com.estelapasjak.Mdots/documentation/MDots/AuthError":{"kind":"symbol","abstract":[{"type":"text","text":"An enumeration representing various authentication errors that can occur."}],"identifier":"doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots\/AuthError","title":"AuthError","role":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"text":"AuthError","kind":"identifier"}],"navigatorTitle":[{"kind":"identifier","text":"AuthError"}],"url":"\/documentation\/mdots\/autherror","type":"topic"}}}